airbyte-webapp-e2e-tests/cypress/pages/settingsPage.ts

```diff
@@ -0,0 +1,33 @@
+export const goToSettingsSourcesTab = () => {
+  cy.intercept("/api/v1/source_definitions/list_for_workspace").as("getSourceDefinitionsList");
+  cy.visit("/settings/source");
+  cy.wait(5000);
+};
+
+export const goToSettingsDestinationTab = () => {
+  cy.intercept("/api/v1/destination_definitions/list_for_workspace").as("getDestinationDefinitionsList");
+  cy.visit("/settings/destination");
+  cy.wait(5000);
+};
+
+export const editVersionByConnectorName = (connectorName: string, version: string) => {
+  cy.get("[data-testid='connectorNameCell']")
+    .contains(connectorName)
+    .parents("tr")
+    .find("[data-testid='version-input']")
+    .focus()
+    .type(`{selectall} {backspace} ${version}`, { force: true })
+    .parents("tr")
+    .find("[data-testid='versionButton']")
+    .click({ force: true });
```

Comment:
This hackiness with `.focus()` and `({ force: true })` shouldn't be needed and aren't working anyways... somehow cypress is finding the element but usually (not always) losing it.  There are a _lot_ of re-renders when cypress is going through these steps...